main.py

```diff
@@ -1,39 +1,80 @@
 #!/usr/bin/env python
 """Implements math functions without using operators except for '+' and '-' """
 
-__author__ = "???"
+__author__ = "patrickmartin333"
 
 
 def add(x, y):
     """Add two integers. Handles negative values."""
     # your code here
-    return
+    return x + y
+
+
+print(add(2, 4))
 
 
 def multiply(x, y):
     """Multiply x with y. Handles negative values of x or y."""
-    # your code here
-    return
+    # result = 0
+    # for i in range(y):
+    #     result += add(result, x)
+    #
+    # if x >= 0 or y >= 0:
+    #     return result
+    # else:
+    #     return operator.neg(result)
+
+    # I want to do this.. but can't seem to figure out how to incorporate my add function.. and it doesn't handle negative numbers
+    result = 0
```

Comment:
This is a great way of thinking about multiply.
Lets say we have a function like this `multiply(x, y)`
Now, 2x5 = 10 and -2x5 = -10. This we know.
So long as we use x to be the negative number, we can `add` our way to the correct answer exactly like you have it: `for i in range y, add x to itself`

If the user passes multiply(5, -2), we can simply reassign x and y
`if x > y then x should become y, and y should become x` This way the negative is always passed as x

The trouble comes when both of the numbers are negative (and 0, but that ones easy) Well, a neg x a nex is alway posative, just like a pos x pos. So if we took the abs() of x and y when both are neg, we could reuse this `for i in range` loop.